README.md

```diff
@@ -1,2 +1,19 @@
 # pcdshub.github.io
 Home page for PCDS Python tools
+
+## How to Contribute
+In order to publish the documentation with `GitHub Pages` the generated HTML documentation needs to be kept on the "master" branch of the repository. To work around this the file the conventional Sphinx `.rst` files that instruct the build are kept on branch "source". If you would like to contribute to the documentation:
+
+1. Fork this repository then check it out on your machine
+2. Create a branch from the "source" branch contained within this repository 
+```bash
+# Add the original repository as a remote
+git remote add pcds https://github.com/pcdshub/pcdshub.github.io
+# Fetch all the branches
+git fetch --all
+# Create your own source branch to track the original source
+git checkout --track pcds/source
```

Comment:
For my own reference, what does this do?

Reply:
It will create a branch "source" in your repository that tracks `pcds/source`. That way you get nice messages on your `git status` like `Your branch is up-to-date with 'pcds/source'.`

https://github.com/k88hudson/git-flight-rules#i-want-to-checkout-to-a-remote-branch-that-someone-else-is-working-on